sys/shell/commands/sc_nimble_netif.c

```diff
@@ -90,12 +111,34 @@ static void _on_ble_evt(int handle, nimble_netif_event_t event,
         case NIMBLE_NETIF_ABORT_SLAVE:
             _print_evt("CONNECTION ABORT", handle, addr);
             break;
+        case NIMBLE_NETIF_ACCEPT_STOP:
+            _print_evt("ACCEPT STOP", handle, addr);
         case NIMBLE_NETIF_CONN_UPDATED:
         default:
             /* do nothing */
             break;
     }
 }
+
+static uint8_t _parsephy(const char *phy_str)
+{
+    if (memcmp(phy_str, "1M", 2) == 0) {
+        return NIMBLE_PHY_1M;
+    }
+#if IS_ACTIVE(MODULE_NIMBLE_PHY_2MBIT)
+    else if(memcmp(phy_str, "2M", 2) == 0) {
```

Comment:
```suggestion
    else if (memcmp(phy_str, "2M", 2) == 0) {
```

Maybe uncristufy while at it?

Reply:
done